src/hooks/DAO/useDAOController.ts

```
@@ -29,11 +29,14 @@ export default function useDAOController() {
     !validDaoQueryString.test(addressWithPrefix) ||
     !isAddress(safeAddressStr);
 
+  if (invalidQuery) {
+    throw new Error('Invalid query');
+  }
+
```

Why this @Da-Colon @DarksightKellar ?

I don't think we ahve the appropriate error boundaries set up to catch this, and and "invalid query" in this context just means that the user might have mistyped the `dao` url query param. Do we really want to throw an error here?